sass/DynamicFacet/_DynamicFacetValues.scss

```diff
@@ -36,8 +36,7 @@
 
   .coveo-checkbox-label:hover,
   &.coveo-focused {
-    .coveo-checkbox-span-label,
-    .coveo-checkbox-span-label-suffix {
+    .coveo-checkbox-span-label {
```

Comment:
Why does the label suffix no longer use the same opacity as the label?

Reply:
Because its color is paler than that of the rest of the text and would have too low of a contrast when hovered.